hack/run-operator-tests.sh

```diff
@@ -19,7 +19,7 @@ docker build . \
     -t "kudobuilder/controller:$KUDO_VERSION"
 
 rm -rf operators
-git clone https://github.com/kudobuilder/operators
+git clone --branch "an/rename-operatorversions" https://github.com/kudobuilder/operators
```

Comment:
This will need to be removed before merging - It requires changes on the operator tests